apps/server/swagger/data-api.yml

```diff
@@ -224,35 +224,22 @@
 
 /data/{systemId}:
   delete:
-    summary: Delete Submitted Data by System ID
+    summary: Delete Submitted Data by System ID. Returns an Active Submission with the records to be deleted
     tags:
-      - admin
+      - Admin
     parameters:
       - name: systemId
         in: path
         required: true
         schema:
           type: string
-      - name: dryRun
-        description: If true won't execute the deletion, but will return what will be deleted, default is false.
-        in: query
-        required: false
-        schema:
-          default: false
-          type: boolean
-      - name: comment
-        description: Comment explaining the reason for deletion.
-        in: query
-        required: false
-        schema:
-          type: string
```

Comment:
Removed both `dryRun` and `comment` fields. These are no longer required as Deletion is handled by active submission